Data-Structures/Tree/TreeTraversal.js

```diff
@@ -0,0 +1,30 @@
+/* eslint no-use-before-define: 0 */ // --> OFF
```

Comment:
Why is this exemption necessary?

Reply:
used it for debugging purpose now removed